code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -73,10 +73,11 @@
 	if(content_size > storage_capacity-5)
 		storage_capacity = content_size + 5
 
-/obj/structure/closet/Initialize(mapload)
+/obj/structure/closet/Initialize(mapload, var/no_fill)
```

Comment:
i don't like this double negative variable, but other than that the rest of the pr is fine